guardrails/utils/openai_utils/__init__.py

```diff
@@ -1,29 +1,16 @@
 from openai.version import VERSION
+from .v1 import AsyncOpenAIClientV1 as AsyncOpenAIClient
+from .v1 import OpenAIClientV1 as OpenAIClient
+from .v1 import (
+    OpenAIServiceUnavailableError,
+    get_static_openai_acreate_func,
+    get_static_openai_chat_acreate_func,
+    get_static_openai_chat_create_func,
+    get_static_openai_create_func,
+)
```

Comment:
we can destructure and remove the .v1 .v0 stuff now and use the OpenAI constructs directly.

Reply:
Since those functions do slightly more than just proxy the method it might be safer to leave these in until we remove direct OpenAI support directly (in favor of litellm).  I just did a search on the `get_static_*` methods and they're pretty widely used.